src/Microsoft.DotNet.Build.CloudTestTasks/RunnerScripts/xunitrunner-perf/csvjsonconvertor.py

```diff
@@ -0,0 +1,200 @@
+import csv
+import getopt
+import json
+import os
+import os.path
+import serialobj
+import sys
+sys.path.append(os.getenv('HELIX_SCRIPT_ROOT'))
+import helix.logs
+
+log = helix.logs.get_logger()
+
+def read_csv(csvFile):
+    log.info('Reading '+csvFile)
+    if not os.path.exists(csvFile):
+        raise Exception(csvFile+' could not be found for conversion')
+
+    csvdict = dict()
+    with open(csvFile, 'rb') as csvfile:
+        reader = csv.reader(csvfile)
+        for row in reader:
+            if row[0] not in csvdict:
+                csvdict[row[0]] = dict()
+            else:
+                if row[1] not in csvdict.get(row[0]):
+                    csvdict.get(row[0])[row[1]] = dict()
```

Comment:
I feel like if there is a single row here, this fails horrible, since it will just create the empty csvdict[row[0]], then exit the loop. Throwing all the data out.

I'm almost positive this whole method is pretty badly broken, causing the first 3 rows to be ignored, and then the first 2 with new second columns, and the first 1 with the new third column.

But it still needs commenting (even a little sample of what the input data is expected to look like would go a long way in making this more maintainable)